Signal/src/ViewControllers/ConversationView/Cells/OWSAudioMessageView.m

```diff
@@ -164,7 +164,7 @@ - (CGFloat)bubbleHeight
 
 + (CGFloat)iconSize
 {
-    return 48.f;
+    return kStandardAvatarSize;
```

Comment:
Pulled out a single constant we can use everywhere, since we're trying to be consistent.  Note that there are a number of places that we show avatars at a larger size: new group view, update group view, app settings view, etc.

Reply:
The larger size now uses: `kLargeAvatarSize`.  Note that there are still other sizes in use in the app, e.g. in the conversation header view.